service/controller/resource/tcnp/template/template_main_security_groups.go

```diff
@@ -38,6 +38,16 @@ const TemplateMainSecurityGroups = `
         ToPort: 10301
         CidrIp: {{ .SecurityGroups.ControlPlane.VPC.CIDR }}
       VpcId: {{ .SecurityGroups.TenantCluster.VPC.ID }}
+  GeneralInternalAPIIngressRule:
+    Type: AWS::EC2::SecurityGroupIngress
+    DependsOn: GeneralSecurityGroup
+    Properties:
+      Description: Allow traffic from the TCNP General Security Group to the TCCP Internal API Security Group.
+      GroupId: {{ .SecurityGroups.TenantCluster.InternalAPI.ID }}
+      IpProtocol: tcp
+      FromPort: 443
```

Comment:
If you are certain that this does not break updates or china, then I am fine here.
I am not certain how this exactly works in an update case, but I imagine that it should be fine.

Reply:
I don't realise any problem that adding a enw security group would cause, this is only node Pools release which we dont hav ein china anyway